scripts/inference.py

```diff
@@ -288,6 +288,7 @@ def inference_process(args: argparse.Namespace):
     generator = torch.manual_seed(42)
 
     for t in range(times):
+        print(f"[{t+1}/{times}]")
```

Comment:
Let me explain why I added it. Currently when you run a long running inference to generate a long video, it is impossible to know how far along I am, because the progress bar gets reset for every frame (?) with no information on which one it's processing. (I also explained in the original PR here https://github.com/fudan-generative-vision/hallo/pull/26).

While this print statement is not the perfect solution and I think a better solution is to have a global progress bar instead, this really is needed.

If you don't like it I can take it out for this PR but then will probably create a new issue for that, just letting you know it's not only NOT an unnecessary change but the most necessary change. Really needed for generating long videos (which may take several hours for people with low spec machines, who will be left hanging without knowing how many more hours they need to wait). Let me know.

Reply:
That's ok. thank you :)